src/Translator.php

```diff
@@ -441,7 +446,7 @@ private function outputPullRequestUrls(): void
     {
         $this->log('<info>The following pull-requests were created:</info>');
         foreach ($this->pullRequestUrls as $pullRequestUrl) {
-            $this->log($pullRequestUrl);
+            $this->log("- $pullRequestUrl");
```

Comment:
This is just an little quality of life thing to make copy-pasting all the urls from terminal to the github issue nicer